src/libsync/abstractnetworkjob.cpp

```diff
@@ -103,6 +103,11 @@ void AbstractNetworkJob::setIgnoreCredentialFailure(bool ignore)
     _ignoreCredentialFailure = ignore;
 }
 
+bool AbstractNetworkJob::ignoreCredentialFailure() const
```

Comment:
This function's name is misleading, since it now covers two variables.

Reply:
It is a function that returns whether credential failures are to be ignored.